### PR TITLE
Image

### PR DESCRIPTION
**Pull request by Tejinder15**
# Created Image Component

## Csb link
https://2xwej.csb.app/Components/Image/image.html

image

### CODE REVIEW

**Review by gadgetfather — APPROVED** [review]
good to merge 👍

File: Components/Image/image.html [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+                        sandbox="allow-scripts allow-same-origin">
+                    </iframe>
+                </div>
+            </div>
+            <hr>
+            <div class="content-feature">
+                <h3>Rounded Image</h3>
+                <div class="content-feature-display">
+                    <p>The .circle-img class shapes the image to a circle.
+                    </p>
```
Comment (VishalPatil18) [inline Components/Image/image.html:64]: Code Indentation could be improved here.

File: Components/Image/image.html [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+                    style="width: 100%; height: 93px; border:0; transform: scale(1); overflow:hidden;"
+                    sandbox="allow-scripts allow-same-origin">
+                </iframe>
+            </div>
+            <hr>
+            <div class="content-feature">
+                <h3>Rounded Corners</h3>
+                <div class="content-feature-display">
+                    <p>The .rounded-img class adds rounded corners to an image.
+                    </p>
```
Comment (VishalPatil18) [inline Components/Image/image.html:81]: Code Indentation could be improved here.

File: Components/Image/image.html [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+                    sandbox="allow-scripts allow-same-origin">
+                </iframe>
+            </div>
+            <hr>
+            <footer>
+                <a href="../Card/card.html" class="prev">
+                    <span class="material-icons-outlined">navigate_before</span>Card
+                </a>
+                <a href="../Image/image.html" class="next">Input <span
+                        class="material-icons-outlined">navigate_next</span></a>
```
Comment (VishalPatil18) [inline Components/Image/image.html:99]: Code Indentation could be improved here.
e.g.

```
<a href="../Image/image.html" class="next">
    Input
    <span class="material-icons-outlined">navigate_next</span>
</a>
```

**Review by VishalPatil18 — APPROVED** [review]
- A few indentation suggestions were added.
- You could change the color of this class name that you're displaying to the user in a way that it looks like code using the `<code></code>` tag. But do this in the future, it's not necessary now.
Screenshot 2022-02-03 at 19 04 35


Apart from these the code looks clean and works great. Great work @Tejinder15 👍

**Pull request merged**